Frameworks/Foundation/NSUserDefaults.mm

```diff
@@ -58,6 +59,9 @@ - (instancetype)init {
 
     _registrationDict = [NSMutableDictionary dictionary];
 
+    _synchronizeQueue = [NSOperationQueue new];
```

Comment:
_synchronizeQueue is a StrongId, which will release the object at the appropriate time.

---
In reply to: [69786087](https://github.com/Microsoft/WinObjC/pull/612#discussion_r69786087) [](ancestors = 69786087)